scripts/docker-compose.yml

```diff
@@ -266,6 +266,96 @@ services:
       retries: 5
     extra_hosts:
       - host.docker.internal:host-gateway
+
+  endorser-api-1:
```

Comment:
Numbering was a bit confusing to me (endorser-api-1 uses the endorser_2 values) but understand after reading through the file. Not sure if there's a typical convention for this stuff with Docker Compose or anything.